xiaqianZhang/assignments/fastSlow/lc141/lc141.py

```diff
@@ -0,0 +1,84 @@
+# Problem Statement #
```

Comment:
Nice pseudocode overview and I can see you're typing the solution twice to help with muscle memory! Nice work.